src/hb-blob.cc

```diff
@@ -520,6 +524,33 @@ _hb_mapped_file_destroy (void *file_)
 }
 #endif
 
+#ifdef _PATH_RSRCFORKSPEC
+static int
+_open_resource_fork(const char *file_name, hb_mapped_file_t *file)
+{
+  size_t name_len = strlen (file_name) + sizeof (_PATH_RSRCFORKSPEC);
+  char *rsrc_name = (char *) malloc (name_len);
+  snprintf (rsrc_name, name_len, "%s%s", file_name, _PATH_RSRCFORKSPEC);
```

Comment:
maybe easier to write like
snprintf (rsrc_name, name_len, "%s" _PATH_RSRCFORKSPEC, file_name);

Reply:
I was going to write it this way but felt a bit confusing as to which goes where.

Reply:
snprintf is an overkill indeed, switched to strncpy/strncat.